spec/injection_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'injection'
+
+describe Array do
+
+	describe 'injection' do
+
+	  it { is_expected.to respond_to :injection }
+
+    context 'with a block' do
+
+      it 'can add numbers together' do
+        expect([1,2,3,4].injection{|accumulator, element| accumulator + element}).to eq 10
```

Comment:
Line is too long. [90/80]<br>Space missing after comma.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.